### PR TITLE
Release 4.2.0

### PR DESCRIPTION
**Pull request by YuyaoHu**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

update version to 4.2.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
 # App Center SDK for Unity Change Log
 
+## Version 4.2.0
+
+### AppCenter
+
+#### UWP
+
+* **[Feature]** In the App Center SDK was added support of strong-named signing assemblies for compatibility with applications that use strong-named signing. Read more about [strong-named assemblies](https://docs.microsoft.com/en-us/dotnet/standard/assembly/strong-named).
```
Comment (MatkovIvan) [inline CHANGELOG.md:9]: Not related

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+* **[Fix]** Fix SDK not sending events when application is packed to self-contained single-file executable.
```
Comment (MatkovIvan) [inline CHANGELOG.md:10]: Not related

Comment (YuyaoHu) [reply CHANGELOG.md:10]: So this time the version number should be 4.1.1? @MatkovIvan

Comment (MatkovIvan) [reply CHANGELOG.md:10]: If it's only fixes, so yes

**Pull request closed without merging**